public/locales/en-US/grafana.json

```diff
@@ -1432,13 +1433,16 @@
       "warning-details-button": "Details"
     },
     "resource-table": {
+      "error-library-element-sub": "Library Element {uid}",
```

Comment:
I would check with @grafana/dashboards-squad on this one - but I think these should just be called "Library Panels".

I _think_ the technical term referred to future possible 'library variables', but no idea what's happening with that and whether they would also be library elements.